app/src/main/res/layout/activity_conversation_v2.xml

```diff
@@ -269,6 +270,39 @@
 
     </FrameLayout>
 
+    <FrameLayout
+        android:id="@+id/outdatedGroupBanner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@+id/blockedBanner"
+        android:background="@color/outdated_client_banner_background_color"
+        android:paddingHorizontal="2dp"
+        android:contentDescription="@string/AccessibilityId_outdated_group_banner"
+        android:visibility="gone">
+
+        <TextView
+            android:id="@+id/outdatedBannerTextView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginVertical="@dimen/very_small_spacing"
+            android:gravity="center_horizontal"
+            android:textColor="@color/black"
+            android:layout_marginHorizontal="@dimen/large_spacing"
+            android:textSize="@dimen/tiny_font_size"
+            android:text="@string/groupLegacyBanner"
+            />
+
+        <ImageView
+            android:layout_gravity="end|center_vertical"
+            android:layout_margin="@dimen/small_spacing"
+            android:layout_width="@dimen/medium_spacing"
+            android:layout_height="wrap_content"
+            android:src="@drawable/ic_link_out"/>
+
+    </FrameLayout>
+
```

Comment:
This banner overlaps the content of the conversation so there are bits that re cut off below it. The banner should probably not hide content underneath

Reply:
Have updated the XML